spec.html

```diff
@@ -20693,19 +21551,52 @@ <h1>Runtime Semantics: ClassDefinitionEvaluation</h1>
         1. Perform ! MakeConstructor(_F_, *false*, _proto_).
         1. If |ClassHeritage_opt| is present, set _F_.[[ConstructorKind]] to ~derived~.
         1. Perform ! CreateMethodProperty(_proto_, *"constructor"*, _F_).
-        1. If |ClassBody_opt| is not present, let _methods_ be a new empty List.
-        1. Else, let _methods_ be NonConstructorMethodDefinitions of |ClassBody|.
-        1. For each |ClassElement| _m_ of _methods_, do
-          1. If IsStatic of _m_ is *false*, then
-            1. Let _status_ be PropertyDefinitionEvaluation of _m_ with arguments _proto_ and *false*.
+        1. If |ClassBody_opt| is not present, let _elements_ be a new empty List.
+        1. Else, let _elements_ be NonConstructorElements of |ClassBody|.
+        1. Let _instancePrivateMethods_ be a new empty List.
+        1. Let _staticPrivateMethods_ be a new empty List.
+        1. Let _instanceFields_ be a new empty List.
+        1. Let _staticFields_ be a new empty List.
+        1. For each |ClassElement| _e_ of _elements_, do
+          1. If IsStatic of _e_ is *false*, then
+            1. Let _field_ be ClassElementEvaluation of _e_ with arguments _proto_ and *false*.
           1. Else,
-            1. Let _status_ be PropertyDefinitionEvaluation of _m_ with arguments _F_ and *false*.
-          1. If _status_ is an abrupt completion, then
+            1. Let _field_ be ClassElementEvaluation of _e_ with arguments _F_ and *false*.
+          1. If _field_ is an abrupt completion, then
             1. Set the running execution context's LexicalEnvironment to _env_.
-            1. Return Completion(_status_).
+            1. Set the running execution context's PrivateEnvironment to _outerPrivateEnvironment_.
+            1. Return Completion(_field_).
+          1. Set _field_ to _field_.[[Value]].
+          1. If _field_ is a PrivateElement Record, then
+            1. Assert: _field_.[[Kind]] is either ~method~ or ~accessor~.
+            1. If IsStatic is *false*, let _container_ be _instancePrivateMethods_.
```

Comment:
```suggestion
            1. If IsStatic of _e_ is *false*, let _container_ be _instancePrivateMethods_.
```
Not sure about `_e_`, but it must be IsStatic of something.

Reply:
It is indeed supposed to be `_e_`.